src/services/person.js

```diff
@@ -37,6 +37,17 @@ class PersonService extends Service {
       }
     );
   }
+
+  async countAllWithSearch(search) {
```

Comment:
since both `person.js` and `course.js` have this method and they all extended from service class, we can move this function to service class.

(below are some extra thoughts, it's suitable for this app, but not all apps. Therefore you can implement it if you like, but not have to.)
after you moved this function to service class, you may find countAll and countAllWithSearch are very similar, we can then group these two function together and inside this merged function. we check whether the input parameter `search` exist, if true, do `searchQueryCount`, if false, do `find().countDocuments()`. Then, not finished yet, since we check the search parameter here, we don't need to check it at the controller level.

Reply:
Thanks for your suggestion. It's a good idea. 
I'd like to implement it in this app.